plugins/ci/pkg/ci/helm.go

```diff
@@ -136,26 +161,50 @@ func handleLocalHelmChart(helm models.HelmConfig, baseRepoFolder, tempFolder str
 
 	helmValuesFiles, err := processHelmValues(helm, nil, tempFolder)
 	if err != nil {
-		return err
+		return models.ScanErrorsReportResult{
+			ErrorMessage: err.Error(),
+			ErrorContext: "processing helm values files",
+			Kind:         "HelmChart",
+			ResourceName: helm.Name,
+			Filename:     helm.Name,
+		}
 	}
 	return doHandleLocalHelmChart(helm, baseRepoFolder, helm.Path, helmValuesFiles, tempFolder, configFolder)
 }
 
 func doHandleLocalHelmChart(helm models.HelmConfig, repoPath string, helmPath string, helmValuesFiles []string, tempFolder, configFolder string) error {
 	helmPath = filepath.Join(repoPath, helmPath)
-	_, err := commands.ExecWithMessage(exec.Command("helm", "dependency", "update", helmPath), "Updating dependencies for "+helm.Name)
+	output, err := commands.ExecWithMessage(exec.Command("helm", "dependency", "update", helmPath), "Updating dependencies for "+helm.Name)
 	if err != nil {
-		return err
+		return models.ScanErrorsReportResult{
+			ErrorMessage: fmt.Sprintf("%v: %s", err, output),
+			ErrorContext: fmt.Sprintf("updating dependencies for helm chart %s", helm.Name),
+			Kind:         "HelmChart",
+			ResourceName: helm.Name,
+			Filename:     helm.Name,
+			Remediation:  "Examine the Chart.yaml file for errors in the dependency specification, or invalid dependencies. See also, https://helm.sh/docs/helm/helm_dependency/",
+		}
 	}
 
 	var helmValuesFileArgs []string
 	for _, vf := range helmValuesFiles {
-		helmValuesFileArgs = append(helmValuesFileArgs, "-f", filepath.Join(repoPath, vf))
+		if filepath.IsAbs(vf) {
+			helmValuesFileArgs = append(helmValuesFileArgs, "-f", vf)
+		} else {
+			helmValuesFileArgs = append(helmValuesFileArgs, "-f", filepath.Join(repoPath, vf))
```

Comment:
what's this?

Reply:
That was addressing an error in processing the temporary file name used to write Helm values defined in fairwinds-insights.yaml. This has been separately addressed by another PR from Vitor.